docs/mods/suite/Quests/Developer/Styling/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Theming the quest book
+toc_max_heading_level: 4
+---
+
+Firstly: Themes are just a `.txt` file that tells quests how to look. Most components will use this for their styling. If they don't. Let us know! Not many people use this system
+
+Secondly: Themes are handled by placing a `ftbquests/ftb_quests_theme.txt` file in your resource pack (kubejs works here)
+
+## The fun stuff
+
+### Properties
+
+#### Color
+- `color`: A colo(u)r property is a RGBA colour code but it supports a bunch of things so here goes
```

Comment:
"colo(u)r" isn't really necessary here.  Just pick US or GB english and be consistent :stuck_out_tongue: